src/Functions/Public/resource-service/Set-vROResourceElement.ps1

```diff
@@ -0,0 +1,101 @@
+function Set-vROResourceElement {
+<#
+    .SYNOPSIS
+    Updates a resource element based off the resource ID.    
+
+    .DESCRIPTION
+    Updates a resource element based off the resource ID. 
+
+    .PARAMETER ResourceID
```

Comment:
Please change ResourceID to be Id to be consistent with the other ResourceElement functions.